nixos/modules/hardware/device-tree.nix

```diff
@@ -16,25 +110,40 @@ in {
           '';
         };
 
-        base = mkOption {
```

Comment:
Oops, this went past me, since this is a removed option, well, one could argue renamed, we want to `mkRemovedOptionModule` with a really quick text pointing towards `hardware.deviceTree.kernelPackage` and to look at the 20.03 release notes.

The end-users would otherwise just see that the option doesn't exist, now they would see an explanation as to why and be orientated about what to do.

Reply:
Do we even need this option? In what case would it make sense to use something different from `config.boot.kernelPackages.kernel` here?

Reply:
I have used files from `raspberrypifw` in the past, though that caused a few issues with a mismatched kernel so I wouldn't necessarily recommend it.

Reply:
Keeping it there for now, it can be useful for 3rd party repositories containing overlays.